### PR TITLE
fixed bugged navbar

### PR DESCRIPTION
**Pull request by spidoosho**

### CODE REVIEW

File: mast/templates/login.html [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
-                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive">
                 <span class="navbar-toggler-icon"></span>
             </button>
             <div class="collapse navbar-collapse" id="navbarResponsive">
-                <ul class="navbar-nav ml-auto">
+                <ul class="navbar-nav">
                     <li class="nav-item">
-                        <a class="nav-link" href="#">Home</a>
+                        <a class="nav-link " href="#">Home</a>
```
Comment (peterkorcsok) [inline mast/templates/login.html:14]: Any reason for the whitespace?

**Review by peterkorcsok — COMMENTED** [review]
Just a question, no problem

**Review by peterkorcsok — CHANGES_REQUESTED** [review]
The same issue for registration page

File: mast/templates/login.html (hunk 1 above)
Comment (spidoosho) [reply mast/templates/login.html:14]: no :D

**Review by peterkorcsok — APPROVED** [review]

**Pull request merged**